listenbrainz/mbid_mapping/mapping/mb_metadata_cache.py

```diff
@@ -13,16 +13,17 @@
 BATCH_SIZE = 5000
 
 
-def mark_rows_as_dirty(lb_conn, recording_mbids: List[uuid.UUID], artist_mbids: List[uuid.UUID]):
+def mark_rows_as_dirty(lb_conn, recording_mbids: List[uuid.UUID], artist_mbids: List[uuid.UUID], release_mbids: List[uuid.UUID]):
     """Mark rows as dirty if the row is for a given recording mbid or if it's by a given artist mbid"""
     try:
         with lb_conn.cursor() as curs:
             query = """UPDATE mb_metadata_cache
                                SET dirty = 't'
                              WHERE recording_mbid = ANY(%s)
                                 OR %s && artist_mbids
+                                OR release_mbid = ANY(%s)
```

Comment:
Replace = ANY with IN ? Both are semantically same afaik but we use IN everywhere else.